src/components/apiErrorAlert/ApiErrorAlert-nb.ts

```diff
@@ -0,0 +1,190 @@
+import { ApiErrorCode } from './apiErrorCode';
+
+const messages: { [key in ApiErrorCode]: string } & { [key: string]: string } = {
```

Comment:
Kan kanskje droppe dette nå
```suggestion
const messages: { [key in ApiErrorCode]: string } = {
```
?